host-configs/LLNL/quartz-base.cmake

```diff
@@ -29,10 +29,14 @@ set(ENABLE_OPENMP ON CACHE BOOL "")
 set(ENABLE_PVTPackage ON CACHE BOOL "")
 set(ENABLE_PETSC OFF CACHE BOOL "Enables PETSc." FORCE)
 
+# PYGEOSX
+set(ENABLE_PYGEOSX ON CACHE BOOL "")
+set(Python3_EXECUTABLE /usr/gapps/GEOSX/thirdPartyLibs/python/quartz-gcc-python/python/bin/python3 CACHE PATH "")
```

Comment:
Sometimes we also need to add this line as well for cmake to find everything:

`set(Python3_ROOT_DIR /usr/gapps/GEOSX/thirdPartyLibs/python/quartz-gcc-python/python CACHE PATH "")`

Reply:
This is added now.